src/openzaak/config/tests/test_nlx_configuration.py

```diff
@@ -20,7 +20,7 @@
 
 CURRENT_DIR = Path(__file__).parent
 
-DEMO_DIRECTORY = "https://directory.demo.nlx.io/"
+DEMO_DIRECTORY = "https://demo-directory-api.commonground.acc.utrecht.nl/"
```

Comment:
you could import this from `zgw_consumers.settings` instead of hardcoding it I guess